src/Cake.Common/Tools/DotNetCore/Build/DotNetCoreBuildSettings.cs

```diff
@@ -42,11 +37,6 @@ public sealed class DotNetCoreBuildSettings : DotNetCoreSettings
         public string VersionSuffix { get; set; }
```

Comment:
Maybe version could be something common, know `dotnet pack` accepts & should have it too (haven't all yet but probably publish too).